libs/langchain-community/src/callbacks/handlers/upstash_ratelimit.ts

```diff
@@ -148,7 +148,7 @@ class UpstashRatelimitHandler extends BaseCallbackHandler {
     _name?: string
   ): Promise<void> {
     if (this.tokenRatelimit) {
-      const remaining = await this.tokenRatelimit.getRemaining(this.identifier);
+      const { remaining } = await this.tokenRatelimit.getRemaining(this.identifier);
```

Comment:
May be some way to make this backwards compatible and allow `"^1.1.3 || ^2.0.3"` in deps above

Reply:
@CahidArda I just pushed up a commit allowing those versions